File: src/main/java/com/brian/guess/Game.java
package com.brian.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {

        Random random = new Random();
        int number = random.nextInt(10) + 1;
//        System.out.println("number is : " + number);

        Scanner scanner = new Scanner(System.in);
        int guessNum = 0;
        while (guessNum != number) {
            System.out.print("please enter number : ");
            guessNum = scanner.nextInt();

            if (guessNum > number) {
                System.out.println("highest");
            } else if (guessNum < number) {
                System.out.println("lowest");
            } else {
                System.out.println("V");
            }
        }
    }
}

File: src/main/kotlin/com/brian/guess/Game.kt
package com.brian.guess

import java.util.*
import kotlin.random.Random

fun main(args: Array<String>) {

    var number: Int = Random.nextInt(10) + 1;
    println("number : $number");

    var guessNum: Int = 0;
    while (guessNum != number) {
        print("Please enter number : ");
        guessNum = Scanner(System.`in`).nextInt();
        if (guessNum > number) {
            println("highest");
        } else if (guessNum < number) {
            println("lowest");
        } else {
            println("V");
        }
    }

}